src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,22 @@
-import React from 'react';
+import { Movie } from '../../react-app-env';
+import { MovieCard } from '../MovieCard';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+interface Prop {
+  moviesList: Movie[]
+}
+
+export const MoviesList: React.FC<Prop> = ({ moviesList }) => (
+  <>
+    {moviesList.map((move: Movie) => (
+      <div key={move.imdbId} className="card">
```

Comment:
You didn't add data attribute
<img width="674" alt="image" src="https://user-images.githubusercontent.com/30314756/170996377-4858c230-dc0f-40f1-b3c6-a7979da7d432.png">